resources/assets/js/sticky-headers.js

```diff
@@ -14,19 +14,18 @@ $(function () {
       'position': 'relative',
       'z-index': 900
     });
-    $(window).scroll(
-      function () {
-        const top = headers.parent().offset().top;
-        const left = 15;
-        timeLanes.css({
-          'left': Math.max(0, $(window).scrollLeft() - left) + 'px'
-        });
-        headers.css({
-          'top': Math.max(0, $(window).scrollTop() - top
-                        - 13
-                        + topReference.offset().top)
-                        + 'px'
-        });
+    $(window).scroll(function () {
```

Comment:
Can you try to add a rule that covers this style to the [ESLint rule set](https://github.com/engelsystem/engelsystem/blob/main/.eslintrc.json)?

Reply:
done